detector/src/main/java/com/synopsys/integration/detector/result/ExcludedDetectorResult.java

```diff
@@ -24,6 +24,6 @@
 
 public class ExcludedDetectorResult extends FailedDetectorResult {
     public ExcludedDetectorResult() {
-        super("Detector type was excluded.", "ExcludedDetectorResult");
+        super("Detector type was excluded.", ExcludedDetectorResult.class);
```

Comment:
Then we wouldn't need to do this.

Reply:
what do you mean by this?

Reply:
Sorry, I mean that if we return this.getClass() then we shouldn't need to use this constructor because it would return this class.  When is the result class different from this.getClass()? Is that when we wrap an underlying detectable result?